hack/run-functests.sh

```diff
@@ -1,14 +1,15 @@
 #!/bin/bash
 set -x
 which ginkgo
+ginkgo version
```

Comment:
What are you checking for here?

Reply:
Removed this line which was introduced during debugging.